pages/events/[id].js

```diff
@@ -0,0 +1,48 @@
+import { useRouter } from "next/router";
+import styled from "styled-components";
+
+const StyledList = styled.ul`
+  list-style: none;
+  padding: 0;
+  margin: 0;
+`;
+const StyledListItem = styled.li`
+  text-align: center;
+  margin: 60px;
+`;
+
+export default function EventDetails({ events }) {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const event = events.find((event) => {
+    return event.id === id;
+  });
+
+  if (!event) {
+    return null;
+  }
+  return (
+    <>
+      <button onClick={() => router.back()}>🔙</button>
```

Comment:
Use a link to the homepage for this.

If you link a section of your app to somebody (e.g. [this](https://capstone-project-economic-calendar-git-eventde-d6f522-jakobraap.vercel.app/events/2fd905f08685634272791124427d349a66cab0e3)) and they click the back button, they go back to an empty tab! And they have no way to go to the homepage!

Generally, links are for navigation and buttons are for actions / submitting forms. It's also less work because you don't have to import any router stuff.

Reply:
Will be replaced by navigation bar